src/ride/ride.c

```diff
@@ -6697,8 +6697,8 @@ void game_command_place_ride_entrance_or_exit(int *eax, int *ebx, int *ecx, int
 money32 remove_ride_entrance_or_exit(sint16 x, sint16 y, uint8 rideIndex, uint8 station_num, uint8 flags){
 	rct_ride* ride = GET_RIDE(rideIndex);
 
-	if (!(flags & (1 << 6))){
-		if (RCT2_GLOBAL(RCT2_ADDRESS_GAME_PAUSED, uint8) != 0){
+	if (!(flags & GAME_COMMAND_FLAG_GHOST)){
```

Comment:
True, this is to be expected when optimization is enabled, but it makes the condition more concise, thus slightly shorter line and easier to fit in head.